### PR TITLE
Add PR Template

### PR DESCRIPTION
**Pull request by aojea**
This TEMPLATE is copied from the kubernetes/kubernetes repository just updated the link with the `kind` contributor guidelines.

xref #611

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines:
+   https://kind.sigs.k8s.io/docs/contributing/getting-started
+2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/testing.md
```
Comment (BenTheElder) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is probably not relevant. I'm also fine with letting CI do the heavy lifting 😅

Comment (aojea) [reply .github/PULL_REQUEST_TEMPLATE.md]: removed

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+**Does this PR introduce a user-facing change?**:
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+-->
+```release-note
+
+```
```
Comment (neolit123) [inline .github/PULL_REQUEST_TEMPLATE.md]: do we have tooling for release note handling or is it planned to have it one day?
possibly omit the `**Does this PR introduce a user-facing change?**:` section until them?

Comment (aojea) [reply .github/PULL_REQUEST_TEMPLATE.md]: will remote it https://github.com/kubernetes-sigs/kind/issues/611#issuecomment-501574779

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines:
+   https://kind.sigs.k8s.io/docs/contributing/getting-started
+2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+3. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+4. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
```
Comment (tao12345666333) [inline .github/PULL_REQUEST_TEMPLATE.md]: this sentence does not seem necessary.

Comment (aojea) [reply .github/PULL_REQUEST_TEMPLATE.md]: good catch 😓

**Comment by tao12345666333** [comment]
/lgtm

thanks

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
```
Comment (BenTheElder) [inline .github/PULL_REQUEST_TEMPLATE.md]: this entire block is kubernetes specific

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+1. If this is your first time, please read our contributor guidelines:
+   https://kind.sigs.k8s.io/docs/contributing/getting-started
+2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+3. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**What type of PR is this?**
+> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
```
Comment (BenTheElder) [inline .github/PULL_REQUEST_TEMPLATE.md]: this block is painful to get right _and_ leaves a huge amount of vertical space wasted by commented out lines that aren't relevant.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
+> /kind flake
+
+**What this PR does / why we need it**:
```
Comment (BenTheElder) [inline .github/PULL_REQUEST_TEMPLATE.md]: a normal PR description covers these remaining fields

**Pull request closed without merging**